README.md

```diff
@@ -15,18 +15,18 @@ housed in this repository.
 
 ## Usage
 
+See the [Installing Nextstrain guide][] for how to install the `nextstrain` command.
+
 If you're unfamiliar with Nextstrain builds, you may want to follow our
-[quickstart guide][] first and then come back here.
+[Running a Pathogen Workflow guide][] first and then come back here.
 
 The easiest way to run this pathogen build is using the Nextstrain
 command-line tool:
 
     nextstrain build .
 
-See the [nextstrain-cli README][nextstrain-cli] for how to install the `nextstrain` command.
-
-Alternatively, you should be able to [run the build using `snakemake` within a
-suitably-configured local environment][nextstrain-snakemake].
+<!--Alternatively, you should be able to [run the build using `snakemake` within a
+suitably-configured local environment][nextstrain-snakemake]. --> 
```

Comment:
Minor: you can usually delete instead of commenting out, since version control retains the history already. In this case, it's safe to delete.